deployment/charts/templates/deploy.yaml

```diff
@@ -0,0 +1,95 @@
+{{- if .Values.app.prometheus.enabled }}
+apiVersion: monitoring.coreos.com/v1
+kind: ServiceMonitor
+metadata:
+  name: {{ template "fuel-block-committer.name" . }}-service-monitor
+  labels:
+    release: {{ .Values.app.prometheus.helm_release_name }}
+spec:
+  selector:
+    app: {{ template "fuel-block-committer.name" . }}
+  endpoints:
+    - path: /metrics
+      port: http
+{{- end }}
+---
+kind: Service
+apiVersion: v1
+metadata:
+  labels:
+    app: {{ template "fuel-block-committer.name" . }}
+    chart: {{ template "fuel-block-committer.chart" . }}
+    release: {{ .Release.Name }}
+    heritage: {{ .Release.Service }}
+  name: {{ template "fuel-block-committer.name" . }}-service
+spec:
+  type: NodePort
+  selector:
+    app: {{ template "fuel-block-committer.name" . }}
```

Comment:
```
        app: {{ template "fuel-block-committer.name" . }}
        release: {{ .Release.Name }}
```

This is a better approach, otherwise it will forward traffic to all deployments in the namespace, which will create issues with network routing

Reply:
done